gallery_app/lib/modules/button/screens/button_screen.dart

```diff
@@ -1,10 +1,285 @@
+                    onPressed: () {},
+                    color: Colors.grey,
+                    child: const Text("Material Button"),
+                  ),
+                ),
+              ),
+              SizedBox(
+                width: double.infinity,
+                child: ClipPath(
+                  clipper: MaterialButtonClip(),
+                  child: MaterialButton(
+                    onPressed: () {},
+                    color: Colors.grey,
+                    child: const Text("Raw Material Button"),
+                  ),
+                ),
+              ),
+              const SizedBox(height: 10),
+              const SelectButtonCustom(),
+              IconButton.filled(
+                onPressed: () {},
+                icon: const Icon(Icons.add),
+                color: Colors.white,
+                style: IconButton.styleFrom(backgroundColor: Colors.blue),
+              )
+            ]),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class SelectButtonCustom extends StatefulWidget {
+  const SelectButtonCustom({
+    super.key,
+  });
+
+  @override
+  State<SelectButtonCustom> createState() => _SelectButtonCustomState();
+}
+
+class _SelectButtonCustomState extends State<SelectButtonCustom> {
+  int selectValue = 1;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        OutlinedButton(
+          onPressed: () {
+            setState(() {
+              selectValue = 1;
+            });
+          },
+          style: OutlinedButton.styleFrom(
+            foregroundColor: Colors.blue,
+            backgroundColor: selectValue == 1 ? Colors.blue[100] : Colors.white,
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(0),
+            ),
+          ),
+          child: const Text("Option 1"),
+        ),
+        OutlinedButton(
+          onPressed: () {
+            setState(() {
+              selectValue = 2;
+            });
+          },
+          style: OutlinedButton.styleFrom(
+            foregroundColor: Colors.blue,
+            backgroundColor: selectValue == 2 ? Colors.blue[100] : Colors.white,
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(0),
+            ),
+          ),
+          child: const Text("Option 2"),
+        ),
+        OutlinedButton(
+          onPressed: () {
+            setState(() {
+              selectValue = 3;
+            });
+          },
+          style: OutlinedButton.styleFrom(
+            foregroundColor: Colors.blue,
+            backgroundColor: selectValue == 3 ? Colors.blue[100] : Colors.white,
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(0),
+            ),
+          ),
+          child: const Text(
+            "Option 3",
+          ),
+        )
+      ],
+    );
+  }
+}
+
+class MaterialButtonClip extends CustomClipper<Path> {
```

Comment:
Do not use Material (Flutter's default names), set it according to your custom UI